lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -38,6 +40,26 @@ class AwsCompileScheduledEvents {
               }
               ScheduleExpression = event.schedule.rate;
               State = event.schedule.enabled ? 'ENABLED' : 'DISABLED';
+              Input = event.schedule.input;
+              InputPath = event.schedule.inputPath;
+
+              if (Input && InputPath) {
+                const errorMessage = [
+                  'You can\'t set both input & inputPath properties at the',
+                  'same time for schedule events.',
+                  'Please check the AWS docs for more info',
+                ].join('');
+                throw new this.serverless.classes
+                  .Error(errorMessage);
+              }
+
+              if (Input && typeof Input === 'object') {
+                Input = JSON.stringify(Input);
+              }
+              if (Input && typeof Input === 'string') {
+                // escape quotes to favor JSON.parse
+                Input = Input.replace(/\"/g,'\\"'); // eslint-disable-line
```

Comment:
@eahefnawy eslint is complaining about the escaped quote inside the regexp (/**\"**/g). Do you see another way of bypassing this?